reference/kube-controllers/configuration.md

```diff
@@ -65,7 +65,7 @@ The following environment variables can be used to configure the {{site.prodname
 | `LOG_LEVEL`           | Minimum log level to be displayed. | debug, info, warning, error | info
 | `KUBECONFIG`          | Path to a kubeconfig file for Kubernetes API access | path |
 | `SYNC_NODE_LABELS`    | When enabled, Kubernetes node labels will be copied to Calico node objects. | boolean | true
-| `AUTO_HOST_ENDPOINTS` | When enabled, automatically create a host endpoint for each node. | boolean | false
+| `AUTO_HOST_ENDPOINTS` | When set to enabled, automatically create a host endpoint for each node. | string | disabled
```

Comment:
we should list valid options - i.e., instead of "string" say "enabled, disabled"

See "LOG_LEVEL" above as an example.